rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -117,6 +123,21 @@ def to_exported_dict(
                 exported_dict['asset'] = ''
             if tx_hash is not None:
                 exported_dict['notes'] = f'{evm_explorer}{tx_hash}  ->  {self.notes}'
+            if self.location in EVM_LOCATIONS and database is not None:
+                def _maybe_add_label_with_address(matched_address: re.Match[str]) -> str:
+                    """Adds a label of the address if it exists in a given string"""
+                    chain_address = ChainAddress(
+                        address=string_to_evm_address(matched_address.group(0)),
+                        blockchain=ChainID(self.location.to_chain_id()).to_blockchain(),
+                    )
+                    name = database.get_blockchain_account_label(chain_address)
+                    return f'{chain_address.address} [{name}]' if name else chain_address.address
+                exported_dict['notes'] = re.sub(
+                    # call _maybe_add_label_with_address on each address in the note
+                    pattern=r'\b0x[a-fA-F0-9]{40}\b',
```

Comment:
is the `\b` needed in the regex? Also maybe makes sense to move it as a constant at have it like `EVM_ADDRESS_REGEX`. You don't need to create the string each time

Reply:
Initially, I did it without `/b` but then it also captured partial transaction hashes and Etherscan URLs. So, I had to specify that it should be a separate word.